tridentctl/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM busybox
+
+ARG TRIDENT_VERSION="19.01.0"
+
+RUN set -ex && \
+    wget -q https://github.com/NetApp/trident/releases/download/v${TRIDENT_VERSION}/trident-installer-${TRIDENT_VERSION}.tar.gz && \
+    tar xzvf trident-installer-${TRIDENT_VERSION}.tar.gz && \
+    mv trident-installer/tridentctl /tridentctl && \
+    chmod +x /tridentctl
+
+RUN set -ex && \
+    wget -q https://storage.googleapis.com/kubernetes-release/release/$(wget -O- -q https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl && \
+    mv kubectl /kubectl && \
+    chmod +x /kubectl && \
+    /kubectl version --client
+
+FROM debian
+
+LABEL io.whalebrew.name tridentctl
+LABEL io.whalebrew.config.environment '["KUBECONFIG","HOME"]'
+LABEL io.whalebrew.config.volumes '["~/.kube:$HOME/.kube:ro","~/.minikube:$HOME/.minikube:ro"]'
```

Comment:
I think Whalebrew will expend `${HOME}` to the current user directory https://github.com/whalebrew/whalebrew/blob/master/cmd/run.go#L64
In any case, docker would not expend it to the `${HOME}` environment variable in the container
Chould have something like `LABEL io.whalebrew.config.volumes '["${KUBECONFIG:-~/.kube/config}:/nobody/.kube/config:ro"]'`, dropping need to expose `KUBECONFIG`, and exposing the minimum information required into the container
This also means there is no way, right now, to use Kubernetes authentication plugins executing a command

Reply:
Someone uses a ca file for apiserver in `~/.kube`. In this case, `certificate-authority` field in kubeconfig is sometimes defined as follows:

```yaml
apiVersion: v1
clusters:
- cluster:
    certificate-authority: /home/superbrothers/ca.crt
```

Therefore, in order to use this kubeconfig file, we need to pass HOME environment variable.

Reply:
could it work to instruct `certificate-authority: ./ca.crt` or `certificate-authority: ~/.kube/ca.crt` 
This would simplify the interface to "kubeconfig must reference resources inside its path, referenced as relative path"
Just trying to minimise the assumptions we are making

Reply:
Yes, using relative path is one of the correct answer. However sometime kubeconfig file is generated by third party tools. In this case, I can not control to use relative or not.

Anyway, I think this PR's approach has no problem. If ca file path is defined as relative path is no problem. If ca file is under `$HOME/.kube` and ca file path is defined as absolute path is also worked well.

Reply:
I see, my fear, although I am not familiar with tridentctl, is that it overriding the HOME environment variable is a bit tacky and we may fall into some strange situations of conflict, `getent passwd` pointing to the old path.
Maybe an alternative is to instruct people to edit their packages to bind additional volumes they y need?
https://github.com/whalebrew/whalebrew/issues/47 could help solve that

Reply:
whalebrew just run a container as package, right? What an issue `getent passwd` pointing to the old path have for running tridentctl package? `HOME` environment variable is just environment variable.

I decide to close this PR because we can't reach to agreement. I continue to manage this package in my private repository.

Thanks.